src/helpers/passable_args/index.js

```diff
@@ -1,32 +1,46 @@
 // @flow
 
+/**
+ * The function which runs the validation tests.
+ *
+ * @callback passableCallback
+ * @param {function} pass
+ * @param {function} enforce
+ */
+
+/**
+ * Get Passable configuration settings
+ * specific - whitelist of tests to run
+ * passes - The function which runs the validations
+ * custom - custom validation rules
+ *
+ * @param {Array.<{specific: String[], passes: passableCallback, custom: Object}>} args - arguments for Passable configuration
+ * @return {object} Passable configuration settings
+ */
 function passableArgs(args: PassableArguments) {
 
     let passes: Function,
-        specific: specific,
-        custom: Rules | void;
-
-    if (args.length === 1) {
-        passes = args[0];
-    } else if (args.length === 3) {
-        specific = args[0];
-        passes = args[1];
-        custom = args[2];
-    } else if (args.length === 2) {
-        if (typeof args[1] === 'function') {
-            specific = args[0];
-            passes = args[1];
-        } else {
-            passes = args[0];
-            custom = args[1];
+        specific: specific = [],
+        custom: Rules = {};
+
+    switch (args.length) {
+      case 0:
+        throw new TypeError('Failed to execute `passableArgs`: At least 1 argument required, but only 0 present.')
+
+      case 1: // [passes] = args;
+        if (typeof args[0] != 'function') {
+          throw new TypeError('Failed to execute `passableArgs`: Unexpected ' + typeof args[0]);
```

Comment:
I would also add here that a function is expected, as a hint for the developer debugging it.